sdk/api-learn/implementation-tutorial/tests/appconfig_preparer.py

```diff
@@ -0,0 +1,128 @@
+                    sku='Standard_LRS',
+                    location='westus',
+                    parameter_name='appconfig_account',
+                    resource_group_parameter_name='resource_group',
+                    disable_recording=True,
+                    playback_fake_resource=None,
+                    client_kwargs=None,
+                    random_name_enabled=True):
+        super(AppConfigPreparer, self).__init__(
+            name_prefix,
+            random_name_length,
+            playback_fake_resource=playback_fake_resource,
+            disable_recording=disable_recording,
+            client_kwargs=client_kwargs,
+            random_name_enabled=random_name_enabled
+        )
+        self.location = location
+        self.sku = sku
+        self.resource_group_parameter_name = resource_group_parameter_name
+        self.parameter_name = parameter_name
+        self.appconfig_key = ''
+        self.appconfig_account_name = ''
+        self.appconfig_url = ''
+        self.appconfig_conn_str = ''
+        self.resource_moniker = self.name_prefix
+        self.use_cache = use_cache
+        if random_name_enabled:
+            self.resource_moniker += "appconfigname"
+
+        self.set_cache(use_cache, sku, location)
+
+    def create_resource(self, name, **kwargs):
+        if self.is_live:
+            self.client = self.create_mgmt_client(AppConfigurationManagementClient)
+            group = self._get_resource_group(**kwargs)
+
+            BODY = {
+                'location': self.location,
+                'sku': {
+                    'name': 'Standard',
+                }
+            }
+            result = self.client.configuration_stores.begin_create(
+                group.name,
+                name,
+                BODY
+            )
+
+            conf_store = result.result()
+
+            self.appconfig_url = conf_store.endpoint
+            self.id = conf_store.id
+            self.keys = list(self.client.configuration_stores.list_keys(group.name, name))
+            self.access_key = self.keys[0].id
+            self.secret = self.keys[0].value
+
+            self.appconfig_conn_str = ";".join([
+                "Endpoint={}".format(self.appconfig_url),
+                "Id={}".format(self.access_key),
+                "Secret={}".format(self.secret)
+            ])
+
+            # appconfig_client = AzureAppConfigurationClient.from_connection_string(self.appconfig_conn_str)
+            # config_setting = ConfigurationSetting(
+            #     key="FontColor",
+            #     value="Green"
+            # )
+
+            # returned_config_setting = appconfig_client.set_configuration_setting(config_setting)
+
+            # config_setting = ConfigurationSetting(
+            #     key="Greeting",
+            #     value="Hello World!"
+            # )
+
+            # returned_config_setting = appconfig_client.set_configuration_setting(config_setting)
```

Comment:
Could we remove these commented out portions?

Reply:
Also, could we remove the left over files from the `sdk/training` directory below?